app/poros/score.rb

```diff
@@ -13,6 +13,10 @@ def show(score_id)
     def delete(score_id)
       FlatService.delete_score(score_id)
     end
+
+    def new_score(score_id)
```

Comment:
these method names are starting to get a little confusing, wonder if we should divide this score poro up further, but that might be just as confusing